packages/shared/src/atoms/ChainSwitch/ChainSwitch.style.tsx

```diff
@@ -0,0 +1,32 @@
+import IconButton from '@mui/material/IconButton';
+import { styled } from '@mui/material/styles';
+import { getContrastAlphaColor } from '../../utils';
+
+export const ButtonChainSwitch = styled(IconButton)(({ theme }) => ({
+  color: theme.palette.text.primary,
+  width: '48px',
+  height: '48px',
+  marginLeft: theme.spacing(3),
+  backgroundColor:
+    theme.palette.mode === 'dark'
+      ? theme.palette.alphaLight300.main
+      : theme.palette.white.main,
```

Comment:
I see this piece being repeated in couple of places. Can we make a function for this?

```ts
const getThemeBackgroundColor = (mode: "light" | "dark") => {
    return theme.palette.mode === 'dark'
      ? theme.palette.alphaLight300.main
      : theme.palette.white.main,
}
```

Reply:
Got it.
My major concern was regarding the usage of same code in multiple places.

Seemed like a good case to extract it out to a helper function to avoid duplication and maintain consistency.

Reply:
We have a lot of places where we use `theme.palette.mode === 'dark'` condition, and it's not always about `backgroundColor`. So we need to go with one approach everywhere, whether we create a helper function for all or leave it as it is. I'm not sure about the helper functions though, the current approach is more straight to the point. 🙂